lib/rdoc/store.rb

```diff
@@ -94,11 +94,7 @@ def message # :nodoc:
 
   attr_accessor :path
 
-  ##
-  # The RDoc::RDoc driver for this parse tree.  This allows classes consulting
-  # the documentation tree to access user-set options, for example.
-
-  attr_accessor :rdoc
+  attr_writer :rdoc
```

Comment:
I think `RDoc::Store` only needs `attr_accessor :options`, but we need to consider the initialization order of `rdoc.options` and `store.rdoc`. Maybe we can do it in another followup pull request later.

Reply:
I have a WIP PR for that (which was meant to be part of this PR). I still have a few tests to fix but can open a draft for it after this is merged.